lab_3_generate_by_ngrams/main.py

```diff
@@ -121,6 +177,18 @@ def decode(self, encoded_corpus: tuple[int, ...]) -> Optional[str]:
         In case of corrupt input arguments, None is returned.
         In case any of methods used return None, None is returned.
         """
+        if not isinstance(encoded_corpus, tuple) or len(encoded_corpus) == 0:
```

Comment:
Empty sequences are mapped to False anyway

Reply:
fixed